LW-1700 Update footer text

foochris opened the pull request.

mcstover commented on src/components/WwwFrame/TheFooter.vue:
```diff
@@ -62,9 +62,8 @@
-						Founded in 2005, and based in San Francisco,
-						with offices in Nairobi and staff around the globe.
-						<router-link to="/donate/supportus">Donate to our operating expenses.</router-link>
+						Founded in 2005, and based in San Francisco, with offices in Bangkok, Nairobi, Portland and staff around the globe.
```
@foochris Please drop the content after "...Nairobi," down to the next line. We have a line limit in our linter that is failing this build. (max 120 characters per line, including tabs)

foochris commented:
@mcstover Change updated as requested

mcstover commented:
@foochris cool, once the Travis builds go green you can merge. Jenkins is not fully setup and is just there to nag us lol

foochris commented:
@mcstover Should I merge even though Travis is all red?

mcstover commented:
No, please fix up based on my comments or run ‘nom run lint:css’ to see the
specific error

On Tue, Sep 18, 2018 at 5:57 PM Chris Foo <notifications@github.com> wrote:

> @mcstover <https://github.com/mcstover> Should I merge even though Travis
> is all red?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kiva/ui/pull/314#issuecomment-422609253>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/ABxsZCcVhgxzm0saI9PFfS7IIavZPYThks5ucZZdgaJpZM4WugfL>
> .
>
-- 

*Matthew Stover | Senior Software Engineer*Mobile: 704-763-7624
*Kiva.org <http://kiva.org/>*

foochris commented:
@mcstover I rebased and squashed a commit. And then:

```
chrisf@chrisf-dev-vm-xenial-01$ /kiva/ui [LW-1700] $ npm run lint:css

> kvue@1.0.0 lint:css /kiva/ui
> stylelint "src/**/*.{vue,scss}"

chrisf@chrisf-dev-vm-xenial-01$ /kiva/ui [LW-1700] $
```

mcstover commented:
You can click through to the "Details" link and inspect the console of the Node.js:node or Node.js:8 build and see the errors at the bottom...not sure why the lint command didn't show this to you locally.

We'll catch up tomorrow and checkout out your setup, get some things in place that will help you see errors before hand.

/home/travis/build/kiva/ui/src/components/WwwFrame/TheFooter.vue
  65:85  error  Trailing spaces not allowed  no-trailing-spaces
✖ 1 problem (1 error, 0 warnings)
